pkg/config/leaderelection/leaderelection.go

```diff
@@ -84,14 +84,25 @@ func ToConfigMapLeaderElection(clientConfig *rest.Config, config configv1.Leader
 func LeaderElectionDefaulting(config configv1.LeaderElection, defaultNamespace, defaultName string) configv1.LeaderElection {
 	ret := *(&config).DeepCopy()
 
+	// 1. lock skew tolerance is leaseDuration-renewDeadline == 22s
```

Comment:
I might suggest we call this method `FastLeaderElectionDefaulting` or `CriticalLeaderElectionDefaulting`, since these defaults should only be used for the most critical services.  Are there a set of library-go based operators that are not critical that can tolerate 30s / 3 retry leases?

One more question - why are you trying to have 6 retries (i.e. what reason for 6 vs 3)?

Reply:
> I might suggest we call this method `FastLeaderElectionDefaulting` or `CriticalLeaderElectionDefaulting`, since these defaults should only be used for the most critical services. Are there a set of library-go based operators that are not critical that can tolerate 30s / 3 retry leases?
> 
> One more question - why are you trying to have 6 retries (i.e. what reason for 6 vs 3)?

Now answered in the code comments.